src/main/java/org/fundacionjala/coding/ana/FizzBuzz.java

```diff
@@ -0,0 +1,43 @@
+package org.fundacionjala.coding.ana;
+
+import java.util.StringJoiner;
+
+/**
+ * FizzBuzz his the data of into.
+ *
+ * @author Ana Maria Mamani Zenteno.
+ */
+public class FizzBuzz {
+    private static final int THREE = 3;
+    private static final int FIVE = 5;
+
+    /**
+     * Method for the number of FizzBuzz.
+     *
+     * @param number any value.
+     * @return a String.
+     */
+    public String numberHundred(int number) {
+        StringJoiner numberStr = new StringJoiner("-");
+        for (int i = 1; i <= number; i++) {
```

Comment:
You can split this method in two
one of them given one number will return fizz, buzz, or fizz buzz